source/reference/limitations.txt

```diff
@@ -88,7 +84,8 @@ Sharded Clusters
   - Replica set to sharded cluster.
   - Sharded cluster to replica set.
   - Unequal numbers of source and destination shards.
-
+- The primary shard cannot be :dbcommand:`reassigned <movePrimary>`
```

Comment:
[suggestion] This should probably be a :ref: and not a :dbcommand: because the word "reassigned" is appearing in monospace.

Reply:
There's no link on that page.  Refactored to use  a `movePrimary` command link instead .